### DISCUSSION TITLE
Is submitting a Pull Request for the examples okay?

### DISCUSSION THREAD

**Discussion by Lexevolution** [ggml-org/whisper.cpp#250] [Q&A] [answered]
I was about to submit a small pull request to the main example, however, I saw that all of the examples are added to the `.gitignore`. So now I'm wonering if any additions/tweaks would be wanted in any of the examples.

**Comment by ggerganov** [comment, marked as answer]
All PRs are very welcome - especially examples.
The `.gitignore` is aimed at ignoring the produced binaries.
For example, if you run `make stream` it will produce the binary `stream`. I want the binary to be ignored.

I now realise that it is probably also ignoring the corresponding folder in `examples/`.
This is not intended - simply override with `git add -f ...`.
If somebody knows how to gitignore just in the root folder - let me know.

**Reply by j-f1** [reply]
I believe adding a leading slash (like `/stream`) will only ignore in the root folder.
